releases.json

```diff
@@ -208,6 +208,14 @@
       "0.4.1-1"
     ]
   },
+  "avro-cpp": {
```

Comment:
You will want to add an entry here for `program_names` to house the avrogencpp program